linode/manager
# chore: Clean Up Repo Root

## Description 📝

### Why?
- I'm trying to clean up the root of our repo because it is bloated with lots of random config files
- We have a nice new readme and this will make it easier to see without needing to scroll on Github

## Overview
- Consolidates two eslint files into one
- Deletes `.npmignore` because it is empty and I'm 99% sure it is doing nothing meaningful
- Deletes old changelog tooling because we don't use it and haven't used it in years

## How to test 🧪

- Not much to test here but you can test the eslint still works
  - One way to do this is to create a duplicate variable  in a random component and ensure that your IDE shows an eslint warning
